lib/main.js

```diff
@@ -473,7 +473,9 @@ function ConfigureMathJax() {
   }
   if (MathJaxConfig) {
     adjustArrays(MathJaxConfig);
-    Insert(window.MathJax,MathJaxConfig);
+    // merge the defaults into the user configuration (to sanitize)
+    var currentMathJax = Insert({},window.MathJax);
+    window.MathJax = Insert(Insert({},MathJaxConfig),currentMathJax);
```

Comment:
These lines could be replaced by just

```
   window.MathJax = Insert(Insert({},MathJaxConfig),window.MathJax);
```

(I had originally intended to make the initial `window.MathJax = { ... }` that defines the mathjax-node configuration be a local variable until this point, but since this is inside an `if` statement, that would make things more complicated, so yours is better.)

I would also consider adding

```
   if (MathJaxConfig.extensions) {
     window.MathJax.extensions = window.MathJax.extensions.concat(MathJaxConfig.extensions);
   }
```

right after this, to append any user-provided extensions onto the mathjax-node ones.  I know that there is already `data.extensions` for that, but since the `extensions` array in the configuration is the usual way to do that, it might be good to handle the as well.